app/models/user.rb

```diff
@@ -50,4 +51,8 @@ def self.find_for_auth(attributes)
       create!(attributes)
     end
   end
+
+  def organisation_valid?
```

Comment:
I don't love the name of this...suggestions welcome

Reply:
Maybe if we move this method into the policy object and call it `user_has_valid_organisation?` then it might feel less awkward? I'm assuming we're not going to need it for anything except the `UserMissingOrganisationError` on the policy.

Reply:
Hmm...We need it in both the policy and scope. I think I moved it into the User model as it only needs data about the user, so I pushed the function down as far as possible.